### PR TITLE
Fix Survey edition page

### PR DESCRIPTION
**Pull request by cveneziani**
Update Survey action context:

- Expose the Survey form
- Add missing delegation of `when_manager` to presenter
- Add missing delegation of `slug` to group

### CODE REVIEW

File: app/contexts/surveys/action_context.rb [hunk 1]
```diff
@@ -3,23 +3,28 @@ class ActionContext
...
+    attr_reader :form
+
     delegate :logo_url,       to: :group, prefix: true
     delegate :manager_email,  to: :group, prefix: true
     delegate :name,           to: :group, prefix: true
+    delegate :slug,           to: :group, prefix: true
     delegate :website_url,    to: :group, prefix: true
 
     delegate :when_group_has_website_url, to: :group_presenter
+    delegate :when_manager,               to: :group_presenter
```
Comment (kdisneur) [inline app/contexts/surveys/action_context.rb]: Is it used somewhere?

Comment (cveneziani) [inline app/contexts/surveys/action_context.rb]: Yes, 2 times in [groups/information_header](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/_information_header.html.slim) partial that is called from [surveys/new](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/surveys/new.html.slim) and [surveys/edit](https://github.com/ruby-nord/gemsavvy/blob/fix/survey-edit/app/views/groups/surveys/edit.html.slim)

Comment (cveneziani) [inline app/contexts/surveys/action_context.rb]: Joy of partials :confused:

**Comment by kdisneur** [comment]
👍

**Pull request merged**